src/coreclr/gc/unix/cgroup.cpp

```diff
@@ -213,7 +213,7 @@ class CGroup
         char *mountpath = nullptr;
         char *mountroot = nullptr;
 
-        FILE *mountinfofile = fopen(PROC_MOUNTINFO_FILENAME, "r");
+        FILE *mountinfofile = fopen(PROC_MOUNTINFO_FILENAME, "rb");
```

Comment:
This file doesn't need to be modified either, it is compiled against system headers, not PAL.